flink/src/test/java/org/apache/iceberg/flink/SimpleDataUtil.java

```diff
@@ -259,4 +265,37 @@ public static StructLikeSet actualRowSet(Table table, Long snapshotId, String...
 
     return dataFiles;
   }
+
+  public static Map<Long, List<DataFile>> snapshotToDataFiles(
+      Table table)
+      throws IOException {
```

Comment:
Nit: The method format looks quite strange for me, could we format it looks like the following?

```
  public static Map<Long, List<DataFile>> snapshotToDataFiles(Table table) throws IOException {
```

Reply:
Done